database/migrations/2023_10_16_121201_create_user_details.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('user_details', function (Blueprint $table) {
+            $table->id();
+            $table->integer("user_id")->nullable();;
+            $table->string('primary_phone');
+            $table->string('alternate_phone');
+            $table->string('customer_number')->nullable();
+            $table->string('shipping_user_name')->nullable();
+            $table->string('shipping_last_name')->nullable();
+            $table->string('shipping_address')->nullable();
+            $table->string('shipping_city')->nullable();
+            $table->string('shipping_state')->nullable();
+            $table->string('shipping_zip')->nullable();
+            $table->string('shipping_country')->nullable();
+            $table->string('shipping_phone')->nullable();
+            $table->string('billing_user_name')->nullable();
+            $table->string('billing_last_name')->nullable();
+            $table->string('billing_address')->nullable();
+            $table->string('billing_city')->nullable();
+            $table->string('billing_state')->nullable();
+            $table->string('billing_zip')->nullable();
+            $table->string('billing_country')->nullable();
+            $table->string('billing_phone')->nullable();
```

Comment:
Set the data types for these fields based on the values we insert. For example, if we insert a number in the 'customer_number' field, it should be of the integer data type.